network/network/src/main/java/bisq/network/NetworkService.java

```diff
@@ -92,7 +92,7 @@
 @Slf4j
 public class NetworkService implements PersistenceClient<NetworkServiceStore>, Service {
     public static final ExecutorService NETWORK_IO_POOL = ExecutorFactory.newCachedThreadPool("Network.IO", 5, 3000, 5);
-    public static final ExecutorService NETWORK_IO_LOW_PRIORITY_POOL = ExecutorFactory.newCachedThreadPool("Network.IO.Low-Priority", 2, 4, 5, ExecutorFactory.LOW_PRIORITY);
+    public static final ExecutorService NETWORK_IO_LOW_PRIORITY_POOL = ExecutorFactory.newCachedThreadPool("Network.IO.Low-Priority", 5, 2000, 5, ExecutorFactory.LOW_PRIORITY);
```

Comment:
I think we can lower the upper limits for the NETWORK_IO_POOL and NETWORK_IO_LOW_PRIORITY_POOL.

It was increased from 300 to 3000 at commit hash: `a5109c828ac20dccdb80a718a548220219c73821`
(Commit msg: `Increase poolsize. There is some threading issue for long running nodes like seeds. This is a temporary mitigation until the cause of the issue is resolved.`)

If you need a higher limit for stress testing, you could add a config parameter to define it.